std/_util/os.ts

```diff
@@ -0,0 +1,17 @@
+// This module is browser compatible.
```

Comment:
Copyright notice is missing:
`// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.`

Reply:
Oh sorry, I'll add it